scripts/ghidra/PatchestryDecompileFunctions.java

```diff
@@ -156,6 +203,167 @@ private HighVariable lValueOf(Varnode node) throws Exception {
         	return var;
         }
 
+        private String getDisplayName(DataType data_type) throws Exception {
+            return data_type.getDisplayName().replaceAll(" ", "");
+        }
+
+        private void serializePointerType(Pointer ptr) throws Exception {
+            name("name").value(getDisplayName(ptr));
```

Comment:
Remove? I don't think we need a name for pointer types. I also don't know why we'd strip out the spaces.

Reply:
Name may not matter with pointer types but having it make it convenient. It will also be consistent with other types: if a pointer is `long****` or `int*`. 

I am stripping out space from display name because I see whitespaces in name like: `long * * * *`. Ideally there should not be whitespace in display name so stripping them.

Reply:
Why is whitespace in the display name a problem, though? Like, if you had `unsigned long *` then you'd make it into `unsignedlong*`.